docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+    "\n",
+    "# Display the figure\n",
+    "display(expdata2.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###  $T_{2}$ v.s  $T_{2}^{\\ast}$\n",
+    "This experiment purpose is to give a better estimation for the dephasing noise. In Ramsey experiment, we can estimate $T_{2}^{\\ast}$ but this is not truly the dephasing noise as the information is not lost.\n",
+    "The $\\ast$ indicates that $T_{2}^{\\ast}$ is sensitive to inhomogeneous broadening. By using echo pulse ($Rx(\\pi)$) we can reduce the effect of the inhomogeneous broadening and get better estimation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "     ┌───┐┌─────────────────┐┌─────────┐ ░ ┌───┐ ░ ┌─┐\n",
+      "  q: ┤ H ├┤ Delay(1e-06[s]) ├┤ Rz(π/5) ├─░─┤ H ├─░─┤M├\n",
+      "     └───┘└─────────────────┘└─────────┘ ░ └───┘ ░ └╥┘\n",
+      "c: 1/═══════════════════════════════════════════════╩═\n",
+      "                                                    0 \n"
+     ]
+    }
+   ],
+   "source": [
+    "import qiskit\n",
+    "from qiskit_experiments.library import T2Ramsey\n",
+    "from qiskit_experiments.test.t2ramsey_backend import T2RamseyBackend\n",
+    "# FakeJob is a wrapper for the backend, to give it the form of a job\n",
+    "from qiskit_experiments.test.utils import FakeJob\n",
+    "\n",
+    "# set the computation units to microseconds\n",
+    "unit_ramsey = \"us\"  # microseconds\n",
+    "qubit_ramsey = 0\n",
+    "# set the desired delays\n",
+    "delays_ramsey = list(range(1, 50, 1))\n",
+    "\n",
+    "conversion_factor_ramsey = 1e-6\n",
+    "# defining backend for the experiment\n",
+    "backend_ramsey = T2RamseyBackend(\n",
+    "    p0={\n",
+    "        \"A\": [0.5],\n",
+    "        \"T2star\": [20.0],\n",
+    "        \"f\": [100100],\n",
+    "        \"phi\": [0.0],\n",
+    "        \"B\": [0.5],\n",
+    "    },\n",
+    "    initial_prob_plus=[0.0],\n",
+    "    readout0to1=[0.02],\n",
+    "    readout1to0=[0.02],\n",
+    "    conversion_factor=conversion_factor_ramsey,\n",
+    ")\n",
+    "\n",
+    "# Create a T2Ramsey experiment. Print the first circuit as an example\n",
+    "exp1_ramsey = T2Ramsey(qubit_ramsey, delays_ramsey, unit=unit_ramsey, osc_freq=1e5, backend=backend_ramsey)\n",
+    "\n",
+    "# Run the experiment\n",
+    "expdata1_ramsey = exp1_ramsey.run(backend=backend_ramsey, shots=2000)\n",
+    "expdata1_ramsey.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Printing a circuit for example\n",
+    "print(exp1_ramsey.circuits()[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that while both backends have $T_2 = 20 [\\mu s]$, the estimation of the Hahn Echo experiment is better."
```

Comment:
Since there's no T1 simulation here, I don't see why to expect T2Hahn to be better than t2Ramsey

Reply:
changed to:
`We can see that the backend has $T_2 = 20 [\mu s]$. We can see the estimate $T_2$ from both experiments:`